docs/en/how-to-loom-js-webpack.md

```diff
@@ -0,0 +1,365 @@
+
+```html
+<html>
+  <head>
+    <title>Set up loom-js with Webpack</title>
+  </head>
+  <body>
+    <script src="./main.js"></script>
+  </body>
+</html>
+```
+
+5. Create a webpack configuration file called `webpack.config.js` with the following content:
+
+```js
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    filename: 'main.js',
+    path: path.resolve(__dirname, 'dist'),
+  },
+};
+```
+
+## The package.json file
+
+1. Open the `package.json` file and replace the `"main": "index.js"` line with `"private": true`.
+
+Open the `package.json` file and add the following line to the `scripts` object:
+
+```json
+"build": "webpack"
+```
+
+At this point you can run `npm run build` to verify if everything works well. You should see something like the following printed out to the console:
+
+```
+Hash: f8f86440ea96399f110d
+Version: webpack 4.41.2
+Time: 110ms
+Built at: 10/25/2019 12:16:21 PM
+ Asset Size Chunks Chunk Names
+main.js 930 bytes 0 [emitted] main
+Entrypoint main = main.js
+[0] ./src/index.js 54 bytes {0} [built]
+
+WARNING in configuration
+The 'mode' option has not been set, webpack will fallback to 'production' for this value. Set 'mode' option to 'development' or 'production' to enable defaults for each environment.
+You can also set it to 'none' to disable any default behavior. Learn more: https://webpack.js.org/configuration/mode/
+```
+
+Notice the warning. In the next section, we'll properly set up the development environment.
+
+## Setting up the Development Environment
+
+Edit the `webpack.config.js` file and make the following changes:
+
+
+1. At the top of the file, add a line that imports the `path` module:
+
+```js
+const path = require('path');
+```
+
+2. Replace the `module.exports` section with the following:
+
+```js
+module.exports = {
+  mode: 'development',
+  devtool: 'inline-source-map',
+  entry: './src/index.js',
+  output: {
+    filename: 'main.js',
+    path: path.resolve(__dirname, 'dist'),
+  },
+}
+```
+
+## The Webpack development server
+
+1. Firt, we must install the Webpack development server with:
+
+```js
+npm install --save-dev webpack-dev-server
+```
+
+2. Now that the required package is installed, let's enable the Webpack development server by adding the following lines to the `module.exports` section of the `webpack.config.js` file:
+
+```js
+devServer: {
+  contentBase: './dist',
+},
+
+```
+
+3. For convenience, we would want to start the web server using `npm run start`. To do so, add the following line to the `scripts` section of the `package.json` file:
+
+```js
+"start": "webpack-dev-server --open"
+```
+
+## Tweaking the webpack.config.js file
+
+1. Because we want to run this in the browser, we must add the following snippet to the `module.exports` section of the `webpack.config.js` file:
+
+```js
+node: {
+ fs: 'empty',
+ child_process: 'empty'
+}
+```
+
+This way, we'll get rid of the compilation errors.
```

Comment:
fs and child_process are node modules, so we need to ignore them while building for the browser. This way, we'll get rid of....

Reply:
@redben Thanks! It's updated.